manifests/params.pp

```diff
@@ -32,11 +32,7 @@
   $ldap_return_attributes  = []
 
   $slf4j_version           = '1.7.22'
-  $slf4j_checksum_type     = 'md5'
```

Comment:
why remove these as parameters?

If you do you may want to list this as a breaking change in the changelog

Reply:
using checksum_url in archive in lieu of